bin/test_module

```diff
@@ -56,15 +56,33 @@ docker_args+=" -v $module_dir/scans:/scans"
 echo Capturing fake startup.pcap...
 sudo tcpdump -c 3 -eni $faux_intf -Z root -w $module_dir/scans/startup.pcap || true
 
+echo Staring module pcap... 
+sudo tcpdump -U -eni $faux_intf -Z root -w $module_dir/scans/test_$module.pcap &
+sleep 5
+echo Module pcap started
+
 docker_args+=" --env TARGET_IP=$faux_ip"
 docker_args+=" --env TARGET_MAC=01:02:03:04:05:06"
 
+
 mkdir -p $config_dir
 docker_args+=" -v $config_dir:/config/inst"
 docker_args+=" -v $config_dir:/config/port"
 docker_args+=" -v $config_dir:/config/device"
 docker_args+=" -v $config_dir:/config/type"
 
+# If testing the tls module, we need a few
+# additional things setup
+if [[ "$module" == "tls" ]]; then
```

Comment:
this doesn't need to be conditional.  Does it hurt anything if the .pem file is always provided?

Reply:
It might start causing failures if it's not checked because it would never resolve anything unless it was the tls module.  This is due to the nature of the start_faux only calling generate_certs.py when the tls module is detected.  The same module detection method was used here to keep that specific scenario in sync.

Reply:
fair enough. I think the bigger point is the one about how we shouldn't be extracting this from the faux device, I think.

Reply:
If we move the cert generation to be available outside of the faux device, then we can update this at the same time.  I've responded on that topic so we can resolve that question there and then double back to this one after that's figured out.